src/connector/postgres.rs

```diff
@@ -223,9 +225,15 @@ impl PostgresUrl {
         let mut host = None;
         let mut socket_timeout = None;
         let mut connect_timeout = None;
+        let mut pg_bouncer = false;
 
         for (k, v) in url.query_pairs() {
             match k.as_ref() {
+                "pgbouncer" => {
+                    pg_bouncer = v
+                        .parse()
+                        .map_err(|_| Error::builder(ErrorKind::InvalidConnectionArguments).build())?;
+                }
```

Comment:
Does this check for `true`/`false` or just presence of `pgbouncer´? (Me no Rust...)

Reply:
true/false

Reply:
so `pg_bouncer` variable is a boolean, `parse` will parse whatever type is on the left side from the string.